ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -148,6 +149,7 @@ void handleResponse(ResponseInfo responseInfo) {
     int correlationId = ((GetRequest) responseInfo.getRequest()).getCorrelationId();
     // Get the GetOperation that generated the request.
     GetRequestInfo getRequestInfo = correlationIdToGetRequestInfo.remove(correlationId);
+    routerMetrics.routerRequestLatencyMs.update(time.milliseconds() - getRequestInfo.startTimeMs);
```

Comment:
This is the latency for a request. The start time is set at the time the request is handed over to the `NetworkClient`. The end time is set at the time the request is received from the `NetworkClient`. That way I think this is the right place.